modules/apps/on-demand-admin/on-demand-admin-test/src/testIntegration/java/com/liferay/on/demand/admin/internal/search/spi/model/permission/test/UserModelPreFilterContributorTest.java

```diff
@@ -38,34 +41,45 @@ public class UserModelPreFilterContributorTest {
 	@ClassRule
 	@Rule
 	public static final AggregateTestRule aggregateTestRule =
-		new LiferayIntegrationTestRule();
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
 
 	@Test
 	public void testSearchUsers() throws Exception {
-		Company company = CompanyTestUtil.addCompany();
+		_company = CompanyTestUtil.addCompany();
 
-		OnDemandAdminTestUtil.addOnDemandAdminUser(company);
-		OnDemandAdminTestUtil.addOnDemandAdminUser(company);
-		UserTestUtil.addUser(company);
-		UserTestUtil.addUser(company);
+		OnDemandAdminTestUtil.addOnDemandAdminUser(_company);
+		OnDemandAdminTestUtil.addOnDemandAdminUser(_company);
+		UserTestUtil.addUser(_company);
+		UserTestUtil.addUser(_company);
 
 		BaseModelSearchResult<User> baseModelSearchResult =
 			_userLocalService.searchUsers(
-				company.getCompanyId(), null, WorkflowConstants.STATUS_APPROVED,
+				_company.getCompanyId(), null,
+				WorkflowConstants.STATUS_APPROVED,
 				new LinkedHashMap<String, Object>(), QueryUtil.ALL_POS,
 				QueryUtil.ALL_POS, new Sort("userId", false));
 
 		int companyUsersCount = _userLocalService.getCompanyUsersCount(
-			company.getCompanyId());
+			_company.getCompanyId());
 
 		Assert.assertEquals(
 			companyUsersCount - 2, baseModelSearchResult.getLength());
 
 		for (User user : baseModelSearchResult.getBaseModels()) {
 			Assert.assertFalse(_onDemandAdminManager.isOnDemandAdminUser(user));
 		}
+
+		_companyLocalService.deleteCompany(_company.getCompanyId());
```

Comment:
is this needed if @DeleteAfterTestRun is used?

Reply:
not needed I left it over